Cooper.ORM/Cooper.ORM.csproj

```diff
@@ -14,4 +14,10 @@
     <ProjectReference Include="..\Cooper.Services.Interfaces\Cooper.Services.Interfaces.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.Logging.Abstractions">
+      <HintPath>C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.extensions.logging.abstractions\2.2.0\lib\netstandard2.0\Microsoft.Extensions.Logging.Abstractions.dll</HintPath>

```

Comment:
Teamcity will say you "hello" when it will try to open Program files. 
And nuGet will send a meme "Am I a joke to you?")